docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,424 @@ HTTP Response
       "status": 200
     }
 
+
+Creating a new User
+********************
+
+This endpoint is a JSON API proxy for the Ledger API's :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>`. For more information about user management, please refer to :ref:`Provisioning Identifiers <provisioning-ledger-identifiers>` part of the Ledger API documentation.
```

Comment:
I did not come up with a suitable replacement for the `provisioning identifiers` part yet. If one of you got an idea please tell me here :)